runtests.jl

```diff
@@ -13,32 +13,37 @@ macro test_broken(ex)
     @test eval(current_module(), ex)
 end
 
-for (root, dirs, files) in walkdir("exercises")
-    for exercise in dirs
-        # Allow only testing specified execises
-        if !isempty(ARGS) && !(exercise in ARGS)
-            continue
-        end
+if !isdir("tmp/")
```

Comment:
In L32 a temp directory is created for each exercise, this shouldn't be needed or am I missing something?

Reply:
It might not be anymore, actually. When I was messing with the `walkdir` variation of this the temp directory was being created within `exercises`, and then the code would try to recurse into the temp dir. Let me try this again without it (and also, what on earth was I thinking with the indentation here?)